{"impact": {"pr": null, "a": "COMPLETE", "ac": "LOW", "impact_score": null, "i": "COMPLETE", "cvss3_score": null, "c": "COMPLETE", "vector_string": "AV:N/AC:L/Au:N/C:N/I:N/A:P", "cvss2_severity": "MEDIUM", "cvss3_severity": null, "s": null, "ui": null, "av": "LOCAL", "cvss2_score": 5.0, "au": "SINGLE", "exploitability_score": null}, "license": "BSD-3-Clause", "name": "django", "cve_id": "CVE-2013-1443", "repository_summary": "A high-level Python Web framework that encourages rapid development and clean, pragmatic design.", "ochrona_severity_score": 5.0, "repo_url": "https://github.com/django/django", "latest_version": "2.2.6", "cwe_id": "CWE-287", "language": "python", "publish_date": "2013-09-23T20:55Z", "references": ["http://lists.opensuse.org/opensuse-updates/2013-10/msg00015.html", "http://lists.opensuse.org/opensuse-updates/2013-11/msg00035.html", "http://python.6.x6.nabble.com/Set-a-reasonable-upper-bound-on-password-length-td5032218.html", "http://www.debian.org/security/2013/dsa-2758", "https://www.djangoproject.com/weblog/2013/sep/15/security/"], "owner": "djangoproject", "affected_versions": [{"operator": "=", "version_value": "1.4"}, {"operator": "=", "version_value": "1.4.1"}, {"operator": "=", "version_value": "1.4.2"}, {"operator": "=", "version_value": "1.4.4"}, {"operator": "=", "version_value": "1.4.5"}, {"operator": "=", "version_value": "1.4.6"}, {"operator": "=", "version_value": "1.4.7"}, {"operator": "=", "version_value": "1.5"}, {"operator": "=", "version_value": "1.5.1"}, {"operator": "=", "version_value": "1.5.2"}, {"operator": "=", "version_value": "1.5.3"}, {"operator": "=", "version_value": "1.6"}], "description": "The authentication framework (django.contrib.auth) in Django 1.4.x before 1.4.8, 1.5.x before 1.5.4, and 1.6.x before 1.6 beta 4 allows remote attackers to cause a denial of service (CPU consumption) via a long password which is then hashed."}